server/views/pages/visit/summary.njk

```diff
@@ -67,22 +68,63 @@
         {% set cancelButtonClasses = 'govuk-!-margin-top-5' %}
         {% if updateJourneyEnabled %}
           {% set cancelButtonClasses = 'govuk-!-margin-top-5 govuk-button--secondary' %}
+
+          {% if visit.visitStatus === 'BOOKED' %}
+            {{ govukButton({
+              text: "Update booking",
+              classes: "govuk-!-margin-top-5",
+              href: "/visit/" + visit.reference + "/update/select-visitors",
+              attributes: { "data-test": "update-visit" },
+              preventDoubleClick: true
+            }) }}
+          {% else %}
+            {{ govukButton({
+              text: "Update booking",
+              classes: "govuk-!-margin-top-5",
+              attributes: { "data-test": "update-visit" },
+              preventDoubleClick: true,
+              disabled: true
```

Comment:
You can just put the condition on a single button. if it's disabled then you can't use it anyway so all you need to do it set `disabled` based on `visit.visitStatus === 'BOOKED'` so could do `disabled: visit.visitStatus === 'BOOKED'`?

Reply:
Yeah - we tried that originally and it _seemed_ to work but it only visually disabled them (i.e. greyed out with CSS). You could still click on them! Found out because we couldn't figure out why the tests we'd written weren't working 🤦‍♂️!